src/GafferScene/Constraint.cpp

```diff
@@ -1083,16 +1155,20 @@ void Constraint::hashProcessedTransform( const ScenePath &path, const Gaffer::Co
 	hashConstraint( context, h );
 }
 
-Imath::M44f Constraint::computeProcessedTransform( const ScenePath &path, const Gaffer::Context *context, const Imath::M44f &inputTransform ) const
+Imath::M44f Constraint::computeConstrainedTransform( const Gaffer::Context *context ) const
 {
+	const M44f inputTransform = inPlug()->transformPlug()->getValue();
+
 	auto targetOpt = target();
 	if( !targetOpt )
 	{
 		return inputTransform;
 	}
 
-	ScenePath parentPath = path;
-	parentPath.pop_back();
+	const Target target = this->target().value();
```

Comment:
Unless I'm missing something, this is unused ( targetOpt is used instead ).

Reply:
Good catch - fix squashed into https://github.com/GafferHQ/gaffer/pull/4848/commits/a41597f801cfecbf08c36e34a38684943a4c432f.